softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/ResponseStatus.java

```diff
@@ -20,9 +20,9 @@
     NOT_IMPLEMENTED(501,"not implemented");
 
     private Integer code;
-    private String message;
+    private Object message;
```

Comment:
@AlexKazmirchuk Object? O_O Message is String. It's written in API. Next - using Object is extremely bad practice!